e2e/cypress/integration/team_settings/closed_team_invite_with_specific_domain_spec.js

```diff
@@ -0,0 +1,141 @@
+
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @team_settings
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import {getEmailUrl, getEmailMessageSeparator, reUrl, getRandomId} from '../../utils';
+
+describe('Team Settings', () => {
+    let testTeam;
+    const randomId = getRandomId();
+    const username = `user${randomId}`;
+    const email = `user${randomId}@sample.mattermost.com`;
+    const password = 'passwd';
```

Comment:
Like I commented at https://github.com/mattermost/mattermost-webapp/pull/6247/files#r475372040, it would be nice to export `generateRandomUser` then import here.